### PR TITLE
Update projects

### PR DESCRIPTION
**Pull request by strickvl**
Updated all projects to more or less latest versions and checked that they worked etc.

### CODE REVIEW

File: customer-satisfaction/requirements.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@ optuna==2.10.0
 streamlit==1.8.1
 xgboost==1.5.2
 markupsafe==1.1.1
-zenml==0.20.5
\ No newline at end of file
+zenml==0.31.1
```
Comment (safoinme) [inline customer-satisfaction/requirements.txt:8]: same as above

**Review by safoinme — APPROVED** [review]
Awesome! Thank you so much for this.
I left a small comment regarding the zenml versions, tho I think it's ready.

**Comment by strickvl** [comment]
> Awesome! Thank you so much for this. I left a small comment regarding the zenml versions, tho I think it's ready.

Given what happened last time with things breaking with the version upgrades, I wouldn't be comfortable releasing it without testing again with the latest version(s).

**Comment by htahir1** [comment]
Eventually we should just add integration tests

**Pull request merged**